### ISSUE TITLE
France - Recovered - very outdated

### ISSUE THREAD

**Issue by bumbeishvili** [CSSEGISandData/COVID-19#1169]
I am merging several datasets to display live data on [my dashboard](https://coronavirus.davidb.dev/)

I noticed that France's recovery data is consistently wrong on every single day (I am using repository data to show the past )

Please correct this 

![Screen Shot 2020-03-21 at 16 13 42](https://user-images.githubusercontent.com/6873202/77226080-ff3df500-6b8e-11ea-8cc9-9d44fd27bafa.png)

**Comment by ecvideo** [comment]
Oui, c'est quoi cett erreur? Elle vient d'où du jeu de donnée ou bien du programme?
Merci de donner des pistes ou chercher dans le mille feuille.
Merci

**Comment by Bost** [comment]
> Oui, c'est quoi cett erreur? Elle vient d'où du jeu de donnée ou bien du programme?
> Merci de donner des pistes ou chercher dans le mille feuille.
> Merci

ENGLISH please!

**Comment by bumbeishvili** [comment]
> Oui, c'est quoi cett erreur? Elle vient d'où du jeu de donnée ou bien du programme?
> Merci de donner des pistes ou chercher dans le mille feuille.
> Merci

![Screen Shot 2020-03-21 at 18 19 06](https://user-images.githubusercontent.com/6873202/77228389-7af46d80-6ba0-11ea-99df-20cabd278458.png)

**Comment by BobCashStory** [comment]
Please use existing issue to make it bigger : https://github.com/CSSEGISandData/COVID-19/issues/1103

**Comment by jcavfen9** [comment]
A ma connaissance les autorités françaises ne publient pas cette donnée. Le site [Santé Publique France](https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde) qui est censé informer le public ne la donne pas. JHU n'est donc pour rien dans cette anomalie et je pense qu'il vaut mieux s'adresser directement au pouvoir public français pour avoir cette donnée en Open Data. Les données détaillées par départements, classes d'âge, sévérités seraient également extrêmement utile. L'écart de transparence par rapport à l'[Italie qui publie les données sur GitHub](https://github.com/pcm-dpc) est criant. [Mon dashboard ](https://public.tableau.com/views/Coronavirus2019-nCoVstats/Coronavirus2019-nCoV?:display_count=y&:origin=viz_share_link ) avec les données du JHU un peu nettoyé pour la France si cela peut vous être utile.

**Comment by jcavfen9** [comment]
I think the problem is coming from French authority reporting not from JHU.

**Comment by bumbeishvili** [comment]
@BobCashStory  I am using that data to display **current** insights

But the past is also important, and that gap may cause misleading assumptions 

@jcavfen9  I am already combining data from several sources and it's hard to maintain it, so I don't want to add another source. It's much better if data will be fixed in this source

**Comment by JiPiBi** [comment]
@jcavfen9 
Your affirmation is not true . The data are given from time to time by Sante Publique 
Why does worldometers has the same data than those given by french authorities 
If you want reliable data on France , and data at a region level go on  https://github.com/opencovid19-fr/data

**Comment by jcavfen9** [comment]
> Your affirmation is not true . The data are given from time to time by Sante Publique
> Why does worldometers has the same data than those given by french authorities ?

Merci pour la source github vers lequel vous m'avez guidé. C'est une super initiative privée. Je n'avais pas vu ces données fournies de manière complète et structurée sur le site [Santé Publique](https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde) ou aucun site gouvernemental.

**Comment by Koesters** [comment]
https://github.com/opencovid19-fr/data/blob/master/ministere-sante/2020-03-20.yaml

National data :
  casesConfirms : 12612
  deaths : 450  # 87% of deaths are people over 70
  hospitalized : 5226  # 35% over 65
  resuscitation : 1297
  cured : 1587

Has cured (Recovered) data, the other files seem to stop at resuscitation.

**Comment by jcavfen9** [comment]
> cured : 1587

Thank you, I have updated [my global dashboard](https://public.tableau.com/views/Coronavirus2019-nCoVstats/Coronavirus2019-nCoV?:display_count=y&publish=yes&:origin=viz_share_link).

**Comment by JiPiBi** [comment]
As I said cured is not given everyday but from time to time 
I agree that cured are given on 17 18 19 20.  in yaml files 
In the site , in folder dist , you have a chifre-cles csv in which the data coming from the yaml are recorded , so chiffres-cles.csv is the main product of the site 
From today, the csv file has hospitalized and cured data at any granularite level when known
